BO4E/ENUM/Status.cs

```diff
@@ -60,6 +60,39 @@ public enum Status
         /// <remarks>Z39</remarks>
         ZAEHLERSTAND_ZUM_ENDE_DER_ANGEGEBENEN_ENERGIEMENGE_NICHT_VORHANDEN_DA_MENGENABGRENZUNG,
 
+        /// <summary> KEIN_ZUGANG </summary>
+        /// <remarks>Z74</remarks>
+        KEIN_ZUGANG,
```

Comment:
wurde in https://github.com/Hochfrequenz/go-bo4e/pull/89 auch ein bestehendes enum verwendet?